tests/integration/linux_bridge_test.py

```diff
@@ -189,7 +211,26 @@ def test_add_port_to_existing_bridge(bridge0_with_port0, port1_up):
     assertlib.assert_state(desired_state)
 
 
-def test_linux_bridge_uses_the_port_mac(port0_up, bridge0_with_port0):
+@pytest.mark.xfail(
```

Comment:
Does it need to be xfail or would skipIf be better given that it is not supposed to be fixed and it would reduce the test load a little bit?

Reply:
Did we agree it should not be fixed?

Reply:
@EdDev What should not be fixed?

Reply:
I was referring to @tyll comment above. He argued it will not be fixed. The logic of using the port mac.

Reply:
I am OK using `skipif(is_fedora())`, so that this test case never kickoff on Fedora 31+ as that's not a bug of Fedora 31 but feature which last forever.
We just use `skipif(is_feadora())` to make sure CentOS 8 does not follow the step of Fedora 31.

@EdDev I still don't get your input on this issue.

Reply:
Changed to `skipif()`. Let me know if you guys think otherwise.